util/config/gather-cray-prgenv-arguments

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+
+THIS_DIR=`dirname "$0"`
+CHPL_HOME="$THIS_DIR/../../"
```

Comment:
It doesn't look like CHPL_HOME is used in this script. Is it needed in the environment for some reason or can it be removed?